File: app/src/main/java/com/example/timemanager/ui/title/ButtonMessage.java
package com.example.timemanager.ui.title;

import android.app.Activity;
import android.content.Context;
import android.content.Intent;
import android.view.View;

import com.example.timemanager.messagePackage.MessageView;


public class ButtonMessage implements View.OnClickListener {
    private Context context;
    private Activity activity;
    public ButtonMessage(Context context, Activity activity){
        this.context = context;
        this.activity = activity;
    }

    @Override
    public void onClick(View view) {
        Intent intent = new Intent(context, MessageView.class);
        activity.startActivity(intent);
    }
}

File: app/src/main/java/com/example/timemanager/ui/title/ButtonMessage.kt
package com.example.timemanager.ui.title

import android.app.Activity
import android.content.Context
import android.content.Intent
import android.view.View
import com.example.timemanager.ui.notifications.MessageView

class ButtonMessage(private val context: Context, private val activity: Activity) :
    View.OnClickListener {
    override fun onClick(view: View) {
        val intent = Intent(context, MessageView::class.java)
        activity.startActivity(intent)
    }

}